renku/api/storage.py

```diff
@@ -132,12 +132,13 @@ def track_paths_in_storage(self, *paths):
                 # TODO create configurable filter and follow .gitattributes
                 track_paths.append(str(path))
 
-        call(
-            self._CMD_STORAGE_TRACK + track_paths,
-            stdout=PIPE,
-            stderr=STDOUT,
-            cwd=str(self.path),
-        )
+        if any(track_paths):
+            call(
```

Comment:
I think this can raise KeyboardInterrupt. Maybe we should handle it and display a nice message to the user instead of a stack trace?